conf/logback.xml

```diff
@@ -31,7 +20,6 @@
   <logger name="com.gargoylesoftware.htmlunit.javascript" level="OFF" />
 
   <root level="INFO">
-    <appender-ref ref="ASYNCFILE" />
```

Comment:
I've removed the `FILE` appenders because the container isn't built with proper permissions for OpenShift at the moment.